web/src/components/organisms/Project/Schema/index.tsx

```diff
@@ -73,56 +53,26 @@ const ProjectSchema: React.FC = () => {
 
   return (
     <>
-      <Layout style={{ minHeight: "100vh" }}>
-        <Header>
-          <MoleculeHeader
-            handleModalOpen={handleWorkspaceModalOpen}
-            personalWorkspace={personalWorkspace}
-            workspaces={workspaces}
-            currentWorkspace={currentWorkspace}
-            user={user}
+      <PaddedContent>
+        <SchemaStyledMenu>
+          <SchemaMenu
+            selectModel={selectModel}
+            defaultSelectedKeys={[model?.id ?? ""]}
+            models={models}
+            handleModalOpen={handleModelModalOpen}></SchemaMenu>
+        </SchemaStyledMenu>
+        <ContentChild>
+          <ModelTitle>{model?.name}</ModelTitle>
+          <ModelFieldList
+            handleFieldUpdateModalOpen={handleFieldUpdateModalOpen}
+            handleFieldDelete={handleFieldDelete}
+            fields={model?.schema.fields}
           />
-        </Header>
-        <Layout>
-          <ProjectSider
-            collapsible
-            collapsed={collapsed}
-            onCollapse={value => setCollapsed(value)}
-            style={{ backgroundColor: "#fff" }}>
-            <ProjectMenu
-              projectId={projectId}
-              defaultSelectedKeys={["schema"]}
-              inlineCollapsed={collapsed}
-              workspaceId={currentWorkspace?.id}
-            />
-          </ProjectSider>
-          <PaddedContent>
-            <SchemaStyledMenu>
-              <SchemaMenu
-                selectModel={selectModel}
-                defaultSelectedKeys={[model?.id ?? ""]}
-                models={models}
-                handleModalOpen={handleModelModalOpen}></SchemaMenu>
-            </SchemaStyledMenu>
-            <ContentChild>
-              <ModelTitle>{model?.name}</ModelTitle>
-              <ModelFieldList
-                handleFieldUpdateModalOpen={handleFieldUpdateModalOpen}
-                handleFieldDelete={handleFieldDelete}
-                fields={model?.schema.fields}
-              />
-            </ContentChild>
-            <FieldListWrapper>
-              <FieldList addField={handleFieldCreationModalOpen} />
-            </FieldListWrapper>
-          </PaddedContent>
-        </Layout>
-      </Layout>
-      <WorkspaceCreationModal
-        open={workspaceModalShown}
-        onClose={handleWorkspaceModalClose}
-        onSubmit={handleWorkspaceCreate}
-      />
+        </ContentChild>
+        <FieldListWrapper>
+          <FieldList addField={handleFieldCreationModalOpen} />
+        </FieldListWrapper>
+      </PaddedContent>
```

Comment:
This file has duplicate FieldUpdateModals. Please delete one

Reply:
@KaWaite , I couldn't find the duplication, please can you explain more?

Reply:
Line 134-151

Reply:
@KaWaite , file is 139 lines long, I think you are viewing the deleted lines?